packages/fxa-event-broker/src/pubsub-proxy/pubsub-proxy.controller.ts

```diff
@@ -32,54 +34,93 @@ export class PubsubProxyController {
   @UseGuards(GoogleJwtAuthGuard)
   @Post(':clientId')
   async proxy(
-    @Res() res: Response,
```

Comment:
When the framework specific Response is passed in, Nest disengages its error handling. To split this I needed the flow control of throwing HTTPExceptions. See more info at https://docs.nestjs.com/controllers#request-object